addons/mail/static/src/models/tracking_value_item.js

```diff
@@ -0,0 +1,111 @@
+             * Field types that are not listed here are not supported by
+             * tracking in Python. Also see `create_tracking_values` in Python.
+             */
+            switch (this.fieldType) {
+                case 'boolean':
+                    return this.value ? this.env._t("Yes") : this.env._t("No");
+                /**
+                 * many2one formatter exists but is expecting id/name_get or data
+                 * object but only the target record name is known in this context.
+                 *
+                 * Selection formatter exists but requires knowing all
+                 * possibilities and they are not given in this context.
+                 */
+                case 'char':
+                case 'many2one':
+                case 'selection':
+                    return format.char(this.value);
+                case 'date':
+                    if (this.value) {
+                        return format.date(moment.utc(this.value));
+                    }
+                    return format.date(this.value);
+                case 'datetime':
+                    if (this.value) {
+                        return format.datetime(moment.utc(this.value));
+                    }
+                    return format.datetime(this.value);
+                case 'float':
+                    return format.float(this.value);
+                case 'integer':
+                    return format.integer(this.value);
+                case 'text':
+                    return format.text(this.value);
+                case 'monetary':
+                    return format.monetary(this.value, undefined, {
+                        currency: this.currencyId
+                            ? this.env.session.currencies[this.currencyId]
+                            : undefined,
+                        forceString: true,
+                    });
+                default :
+                    return this.value;
+            }
+        },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeValueAsString() {
+            return this.formattedValue || this.env._t("None");
+        },
+    },
+    fields: {
+        /**
+         * Used when the currency changes as the tracking value. This only makes sense for field of type monetary.
+         */
+        currencyId: attr(),
+        /**
+         * Indicates the type of the tracking value item.
+         * The supported types are: boolean, char, many2one, selection, date, datetime, float, integer, text, monetary.
+         */
+        fieldType: attr(),
+        /**
+         * The formatted value of the tracking value item.
+         */
+        formattedValue: attr({
+            compute: '_computeFormattedValue',
+        }),
+        /**
+         * States the tracking value displaying the item.
+         * Generally, it is mutually exclusive with trackingValueAsOldValue.
+         */
```

Comment:
not very useful, this should be obvious to anyone understanding how identifying fields work